README.md

```diff
@@ -75,29 +77,43 @@ All files are located under `test/` folder.
 │   │   │   │   │   └── web_addon_test.exs
 ```
 
-#### 2/ Variant test
+### 2/ Variant test
 
-##### 2.1/ Variant
+#### 2.1/ Variant
 
 NimblePhxGenTemplate is supporting 3 variants:  
 
 - API
 - Web
 - Live
 
-##### 2.2/ Phoenix project
+#### 2.2/ Phoenix project
 
 The Phoenix project could be either a Web project or API project.
 
-- Web project is including HTML and Webpack configuration, which was generated by `mix phx.new your_app_name`.
+- Web variant supports HTML and Webpack configuration.
+
+```bash
+mix phx.new your_app_name
+```
+
+- API variant does NOT support HTML and Webpack configuration.
+
+```bash
+mix phx.new your_app_name --no-html --no-webpack
+```
 
-- API project is NOT including HTML and Webpack configuration, which was generated by `mix phx.new your_app_name --no-html --no-webpack`.
+- Custom project variant allow us to modify the app name or module name.
 
-Aside from that, it could also be a Custom project, which contains the custom OTP app name or custom module name.
+```bash
+# Use CustomModuleName
+mix phx.new your_app_name --module=CustomModuleName
 
-- `mix phx.new your_app_name --module=CustomModuleName`
-- `mix phx.new your_app_name --app=custom_otp_app_name`
-- `mix phx.new your_app_name --module=CustomModuleName --app=custom_otp_app_name`
+# Use custom otp app name
+mix phx.new AppName --app=custom_otp_app_name
+# Use custom module and app name
+mix phx.new AppName --module=CustomModuleName --app=custom_otp_app_name
```

Comment:
1/ AppName -> `your_app_name`

2/ I prefer the other name instead of `your_app_name` @byhbt, how about `awesome_project` for example, what do you think? That makes another mood for people :D

Reply:
or just `app_name` short and simpler :D

Reply:
cool idea 👍

Reply:
@byhbt don't forget the `mix phx.new AppName` there :D

Reply:
Good catch, I updated again. Thanks @andyduong1920 !